### PR TITLE
Improve content single screens with design polish

### LINKED ISSUE #275: Content UI/UX polish
## User Story

As an app user I want to be focused on what matters so that I connect with God and grow within my church community.

#### Acceptance Criteria

- [ ] find a home for `share` and `like` buttons
- [ ] clean up loading states
- [ ] add padding to media icons (video/audio)
- [ ] preserve devotional nav scroll position/auto advance
- [ ] etc.

### PR DESCRIPTION
**Pull request by conrad-vanl**
Fixes #275 

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings (in app AND in storybook)
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer
- [ ] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

File: packages/apolloschurchapp/src/content-feed/getContentFeed.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import { contentItemFragment } from '../content-single/getContentItem';
+
 export default gql`
   query getContentFeed($itemId: ID!) {
     node(id: $itemId) {
       ... on ContentChannel {
         childContentItemsConnection {
           edges {
             node {
+              ...contentItemFragment
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/content-feed/getContentFeed.js]: 👍 I totally endorse more usage for fragments.

File: packages/apolloschurchapp/src/content-single/index.js [hunk 2]
```diff
@@ -1,247 +1,95 @@
...
+          <ActionContainer itemId={id} />
+        </ModalView>
+      </ThemeMixin>
+    );
+  };
+
+  render() {
+    return (
+      <Query query={getContentItem} variables={this.queryVariables}>
+        {this.renderWithData}
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/content-single/index.js]: I like using this `renderWithData`. It takes a lot of the visual formatting complexity out of using Query Components

File: packages/apolloschurchapp/src/tabs/discover/TileImageItem.js [hunk 3]
```diff
@@ -22,7 +22,6 @@ const TileImageItem = ({
       onPressItem={() =>
         navigation.navigate('ContentSingle', {
           itemId: id,
-          sharing,
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/tabs/discover/TileImageItem.js]: I'm so glad you took sharing out of the navigate call.

**Comment by vinnyjth** [comment]
@conrad-vanl On smaller devices (iPhone 6) the media bar margin feels a little tight. Is that the correct spacing? 

screenshot 2018-10-12 12 45 30
screenshot 2018-10-12 12 44 43

**Comment by vinnyjth** [comment]
@conrad-vanl Hard to reproduce this but I'll report it...

1. Play a piece of content.
2. Minimize the media player 
3. Pause or player the media player.
 - The warning only shows up the _first_ time you pause or play.

Curiously enough, I was able to reproduce by doing the "inverse" of those steps.

1. Open the mini-player into the full view.
2. Pause or player the media player.
 - The warning only shows up the _first_ time you pause or play

screenshot 2018-10-12 13 28 41

File: packages/apolloschurchapp/src/content-single/HorizontalContentFeed/getHorizontalContent.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+export default gql`
+  query getHorizontalContent($itemId: ID!) {
+    node(id: $itemId) {
+      ... on ContentItem {
+        id
+        childContentItemsConnection {
+          edges {
+            node {
+              id
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/content-single/HorizontalContentFeed/getHorizontalContent.js]: Would it be worth adding this set of attributes to a fragment?

**Comment by conrad-vanl** [comment]
Needs #334

**Comment by conrad-vanl** [comment]
This is ready as soon as #334 is merged

**Comment by conrad-vanl** [comment]
@vinnyjth this is ready now, I believe!!!

The warnings are in master and can be fixed in a sep PR.

**Pull request merged**